src/pruna/algorithms/quantization/quanto.py

```diff
@@ -102,30 +105,32 @@ def _apply(self, model: Any, smash_config: SmashConfigPrefixWrapper) -> Any:
             The quantized model.
         """
         imported_modules = self.import_algorithm_packages()
-
         if hasattr(model, "unet"):
             working_model = model.unet
         elif hasattr(model, "transformer"):
             working_model = model.transformer
         else:
             working_model = model
 
-        weights = getattr(imported_modules["optimum"].quanto, smash_config["weight_bits"])
-        if smash_config["act_bits"] is not None:
-            activations = getattr(imported_modules["optimum"].quanto, smash_config["act_bits"])
-        else:
-            activations = None
+        imported_modules = self.import_algorithm_packages()
+
+        weights = getattr(imported_modules["quanto"], smash_config["weight_bits"])
+        activations = (
+            getattr(imported_modules["quanto"], smash_config["act_bits"])
+            if smash_config["act_bits"] is not None
+            else None
+        )
 
         try:
-            imported_modules["quantize"](working_model, weights=weights, activations=activations)
+            quantize(working_model, weights=weights, activations=activations)
         except Exception as e:
             pruna_logger.error("Error during quantization: %s", e)
             raise
 
         if smash_config["calibrate"]:
             if smash_config.tokenizer is not None and smash_config.data is not None:
                 try:
-                    with imported_modules["Calibration"](streamline=True, debug=False):
+                    with Calibration(streamline=True, debug=False):
```

Comment:
same here, use the imported_modules
```suggestion
                    with imported_modules["Calibration"](streamline=True, debug=False):
```